decentraland/unity-renderer
# Feat: UI loading wheel

## What does this PR change?

This PR removes the loading spinner based on sprites, and replaces it with a spinner made with shader graph. New materials where added for the different uses of the spinners.


...

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/UI-Loading-Wheel
2. Open the main menu
3. Verify the correct behaviour of the loading spinners

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
